### ISSUE TITLE
not working on raspberry pi 2 ?

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [YouTransfer/YouTransfer#177]
Hi,

I just tried to install your very promissing package on my raspberry pi 2 running raspbian after installing `npm` and `nodejs` by doing `PORT=80 npm start` in the folder I got with `git clone https://github.com/YouTransfer/YouTransfer.git` and I get this error : 

>PORT=80 npm start
youtransfer@1.1.0 start /home/USER/YouTransfer
gulp clean && gulp dist && node app.js
sh: 1: gulp: not found
npm ERR! youtransfer@1.1.0 start: `gulp clean && gulp dist && node app.js`
npm ERR! `sh "-c" "gulp clean && gulp dist && node app.js"` failed with 127
npm ERR! 
npm ERR! Failed at the youtransfer@1.1.0 start script.
npm ERR! This is most likely a problem with the youtransfer package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp clean && gulp dist && node app.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls youtransfer
npm ERR! There is likely additional logging output above.
npm ERR! 
npm ERR! System Linux 4.1.19-v7+
npm ERR! command "/usr/bin/nodejs" "/usr/bin/X11/npm" "start"
npm ERR! cwd /home/USER/YouTransfer
npm ERR! node -v v0.6.19
npm ERR! npm -v 1.1.4
npm ERR! code ELIFECYCLE
npm ERR! message youtransfer@1.1.0 start: `gulp clean && gulp dist && node app.js`
npm ERR! message `sh "-c" "gulp clean && gulp dist && node app.js"` failed with 127
npm ERR! errno {}
npm WARN This failure might be due to the use of legacy binary "node" 
npm WARN For further explanations, please read
npm WARN /usr/share/doc/nodejs/README.Debian
npm WARN 
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /home/USER/YouTransfer/npm-debug.log
npm not ok

I am pretty noobish for all that is nodejs and selfhosting related but I had not read anywhere that it should or shouldn't be compatible with a raspberry pi so I wanted to try it. Do you think it isfeasible ? Do you want me to provide any kinds of other infos ?

**Comment by remie** [comment]
@thiswillbeyourgithub: It should definitely work on your Raspberry Pi! However, you will need to run `npm install` first in the installation folder prior to running `npm start`. This will download the dependencies required for the application to work properly.

**Comment by thiswillbeyourgithub** [comment]
I just checked and indeed I missed the `npm install` instruction in the wiki. My bad :(

Okay so I had to update quite a lot using `sudo npm cache clean -f && sudo npm install -g n && sudo n stable` before seeing `npm install` succeed otherwise I had various errors much like this one `failed to fetch from registry : XXX`. Then I had to reboot and finally `sudo PORT=80 npm start` succeeded.

It doesn't work yet but I'll take it from there. Thanks a lot !
